qiskit_ibm_runtime/utils/utils.py

```diff
@@ -99,6 +99,36 @@ def validate_isa_circuits(circuits: Sequence[QuantumCircuit], target: Target) ->
             )
 
 
+def are_circuits_dynamic(circuits: List[QuantumCircuit]) -> bool:
+    """Checks if the input circuits are dynamic."""
+    for circuit in circuits:
+        if isinstance(circuit, str):
+            return True
```

Comment:
We don't currently accept OpenQASM3 as an input circuit type, so this would essentially be a no-op. If/when we do, it shouldn't immediately be treated as containing control flow. 

```suggestion
```